packages/js/components/src/experimental-select-control/stories/index.tsx

```diff
@@ -176,25 +176,35 @@ export const CustomRender: React.FC = () => {
 		setSelected( [ ...selected, item ] );
 	};
 
+	const getFilteredItems = ( allItems: ItemType[], inputValue: string ) => {
+		const re = new RegExp( inputValue, 'gi' );
```

Comment:
I think the `inputValue` needs to be escaped unless we want to allow regex input 😄 

Can either use lodash's `escapeRegExp` or vanilla JS (preferable IMO so we can eventually remove lodash) - `string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');`